Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/covid_priority.cpp

```diff
@@ -0,0 +1,452 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Sight++. All Rights Reserved.
+
+#include "covid_priority.hpp"
```

Comment:
Another general thought, I think ideally this should more generalised so that each priority implementations doesn't have to add ~450 lines of code which is 99% copy-paste 😄